.github/workflows/ci.yml

```diff
@@ -24,6 +24,10 @@ jobs:
             otp: 24.1
             check_formatted: true
             warnings_as_errors: true
```

Comment:
In theory, we want to remove these lines since you moved them to Elixir 1.14. (So that any discrepancies between 1.14 and 1.13's errors and formatting don't make CI fail.)

Reply:
Although in practice it looks like there aren't any discrepancies affecting us.

Reply:
That's a good point though. I get what you're saying. Will fix in the morning